### ISSUE TITLE
Error on building after version 0.9.3

### ISSUE THREAD

**Issue by 5v3n-08** [CodeDredd/pinia-orm#77]
After updating to version 0.9.3, I get the following error message when building:

`ERROR in ./node_modules/@pinia-orm/nuxt/dist/runtime/nuxt2-plugin.mjs
Module not found: Error: Can't resolve 'pinia-orm' in 'xxx/node_modules/@pinia-orm/nuxt/dist/runtime'
 @ ./node_modules/@pinia-orm/nuxt/dist/runtime/nuxt2-plugin.mjs 1:0-33 3:17-25
 @ ./node_modules/.cache/nuxt/index.js
 @ ./node_modules/.cache/nuxt/client.js
 @ multi ./node_modules/@nuxt/components/lib/installComponents.js ./node_modules/.cache/nuxt/composition-api/register.js ./node_modules/.cache/nuxt/client.js

ERROR in ./node_modules/pinia-orm/dist/index.mjs 1485:75
Module parse failed: Unexpected token (1485:75)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|       const type = model[this.morphType];
|       const id = model[this.morphId];
>       const related = ((_a = dictionary[type]) == null ? void 0 : _a[id]) ?? null;
|       model.$setRelation(relation, related);
|     });
 @ ./src/modules/application/storing/storing.service.ts 11:0-36 80:28-35 101:8-15 118:6-13
 @ ./src/modules/application/storing/storing.plugin.ts
 @ ./node_modules/.cache/nuxt/index.js
 @ ./node_modules/.cache/nuxt/client.js
 @ multi ./node_modules/@nuxt/components/lib/installComponents.js ./node_modules/.cache/nuxt/composition-api/register.js ./node_modules/.cache/nuxt/client.js`

I am currently using pinia-orm with nuxt2.

I suspect that the error is in this commit [09e0ebe](https://github.com/CodeDredd/pinia-orm/commit/09e0ebef1f7f15a13b44faddb27d639e25779804), here in line 26 the require was changed to .cjs

**Comment by CodeDredd** [comment]
thats true. thanks for pointing it out...i hoped that the js build would be enough....seems have to make cjs build and mjs build
